Assets/Scripts/HandController.cs

```diff
@@ -4,36 +4,53 @@
 
 public class HandController : MonoBehaviour
 {
+    public List<GameObject> cards;
+    public List<GameObject> selectedCards;
```

Comment:
would we rather have two lists of cards or just have the individual card have a flag that indicates whether or not they are selected?

Reply:
Definitely need to give this more thought.

At first glance, definitely better for memory to not make another List, and the cards already do have selected flags (`CardController.selected`), so we could just iterate on the `cards` array and filter for selected cards.

Down the line once we actually start casting spells using the selected cards, I feel like the flag approach could get unwieldy. Especially if we move the selected cards to another "staging area" and want to allow for reordering etc. In this case, another List would be a natural code simplification that I might argue is worth the memory tradeoff.

Definitely something to revisit. Keep bugging me on this.

Reply:
Sounds good to me! I'll keep banging my head on this.